modules/ROOT/pages/about-components.adoc

```diff
@@ -1,57 +1,176 @@
+* xref:dynamic-evaluate-component-reference.adoc[Dynamic Evaluate]: For dynamically selecting a script, instead of hardcoding it through the Transform Message Component.
+* xref:flowref-about.adoc[Flow Reference]: For routing the Mule event to another
+flow or subflow (and back) within a Mule app.
+* xref:logger-component-reference.adoc[Logger]: For logging important information about
+your Mule app, such as error messages and status notifications.
+* xref:parse-template-reference.adoc[Parse Template]: For processing a template
+and obtaining a result.
+* xref:transform-component-about.adoc[Transform Message]: For converting input
+data to a new output structure or format.
 
-A Scope (blue color in Design Center) groups a sequence of event processors together to apply some programming behavior to that isolated sequence of event processors.
+== Endpoints
 
-For example, a Try scope lets you isolate and handle any errors that occur in a particular sequence of flows. So, you might wrap an outbound HTTP Request connector (and perhaps some other surrounding components before and after the HTTP Request connector, such as a Transform Message component or a Logging component) in a Try scope, so if an error results, you can apply logging or compensation logic specific to that grouping of components.
+An endpoint is a Core component that initiates (or triggers) processing in a
+Mule flow. The xref:scheduler-concept.adoc[Scheduler] serves as an endpoint. It triggers a flow to start at a configurable interval.
 
-A For Each scope takes a collection of data, usually extracted from the current input event, and applies the same sequence of event processors to every item in the collection. For example, a For Each scope might be used to process each individual row returned from a database query or each individual line from a CSV file.
+Note that some connectors provide listeners that serve as endpoints. Listeners
+can trigger a flow when they receive an external request. For example, an HTTP
+Listener that is configured for a given URL can trigger a flow when someone or \
+some process goes to that URL.
+
+== Error Handling
+
+Error handling components route and process Mule errors that occur in a Mule app:
+
+* Error Handler
+* On Error Continue
+* On Error Propagate
+
+See xref:error-handling.adoc[Error Handlers]
+
+== Flow Control (Routers)
+
+A Flow Control takes the input event and routes it to one or more separate sequences of components. Flow Control components (or Routers) include:
+
+* Choice
+* First Successful
+* Round Robin
+* Scatter-Gather
+
+For example, a xref:choice-router-concept.adoc[Choice router]) applies
+DataWeave logic to pick one of several routes, where each route is a separate sequence of event processors.
+
+A xref:scatter-gather-concept.adoc[Scatter-Gather router] router copies the
```

Comment:
I wouldn't use "copy" but rather something like "sends" to avoid confusing people on whether some payload can be copied or not.

Reply:
ok, great, will change that